packages/functionals/botpress-nlu/src/views/style.scss

```diff
@@ -92,6 +95,31 @@ footer {
   }
 }
 
+.intentsContainer {
+  height: 100%;
+}
+
+.intentsList {
+  max-height: calc(100% - 20px);
+  overflow-y: auto;
+
+  &::-webkit-scrollbar {
```

Comment:
Obviously, is WebKit-only. Should we add more vendor prefixes?

Reply:
@emirotin no, because other browsers don't support similar properties

Reply:
I remembered IE had some custom props for that too so I thought other browser may as well but apparently you're right so ignore me :)

Reply:
If I know right IE have color properties)